src/comment.rs

```diff
@@ -300,10 +300,10 @@ mod test {
                                    12));
 
         let input = "// comment";
-        let expected = "/* com\n                                                                      \
-                        * men\n                                                                      \
-                        * t */";
-        assert_eq!(expected, rewrite_comment(input, true, 9, 69));
+        let expt = "/* com\n                                                                      \
```

Comment:
It is. I cheated a bit to fit the line within the limit, but I now realize there's better ways of achieving that.